x-pack/plugins/security_solution/public/timelines/components/timeline/properties/new_template_timeline.test.tsx

```diff
@@ -101,7 +101,7 @@ describe('NewTemplateTimeline', () => {
     });
 
     afterAll(() => {
-      (useKibana as jest.Mock).mockReset();
+      jest.clearAllMocks();
```

Comment:
This is redundant now that the `beforeEach()` is hoisted to the outer describe.

Reply:
Removed in 1116cefa6ae6f9bc1cb3d8aae9c1801d24a10a4f